build/NugetConfigFile.targets

```diff
@@ -20,7 +20,7 @@
         <![CDATA[
     <add key="BlobFeed" value="https://dotnetfeed.blob.core.windows.net/dotnet-core/index.json" />
     <add key="templating" value="https://dotnet.myget.org/F/templating/api/v3/index.json" />
-    <add key="aspnetcore-dev" value="https://dotnet.myget.org/F/aspnetcore-dev/api/v3/index.json" />
+    <add key="aspnetcore-release" value="https://dotnet.myget.org/F/aspnetcore-release/api/v3/index.json" />
```

Comment:
Who owns keeping these in sync?

Reply:
ProdCon and dotnet-maestro-bot...which haven't successfully completed in a while, so Pranav an I have been doing it manually.